### ISSUE TITLE
Documentation missing: How should files in LDAP_CUSTOM_LDIF_DIR look like?

### ISSUE THREAD

**Issue by johanneskastl** [bitnami/bitnami-docker-openldap#83]
### Name and Version

bitnami/openldap:2.5.11

### What steps will reproduce the bug?

I am missing details on how exactly the LDIF files should look like, that one can put into a directory `/ldifs` (or the one set in `LDAP_CUSTOM_LDIF_DIR`).

Especially as the `cn=config` contains a database with `olcSuffix`, does the server take care of setting the proper suffix after importing "my" data? 

Or do I need to additionally set `LDAP_ROOT`? But the docs say:

> When using this will override the usage of `LDAP_ROOT`, `LDAP_USERS`, `LDAP_PASSWORDS`, `LDAP_USER_DC` and `LDAP_GROUP`.

Should I just add another database to the files that are to be imported?

```
# {3}mdb, config
dn: olcDatabase={3}mdb,cn=config
objectClass: olcDatabaseConfig
objectClass: olcMdbConfig
olcDatabase: {3}mdb
olcDbDirectory: /bitnami/openldap/data/mydb
olcSuffix: dc=foo,dc=org

dn: dc=foo,dc=org
objectClass: domain
objectClass: dcObject
...
```

### What do you see instead?

The docs should contain some more details on what to set how and where.

**Comment by johanneskastl** [comment]
Hi @fmulero thanks for the fast reply. OK, so this is only intended for "normal" LDIF files.

In the default configuration, there is a database `dn: olcDatabase={3}mdb,cn=config` that contains a `olcSuffix`. How would I change this, if `LDAP_ROOT` is being ignored? Or is this automatically changed?

(I cannot test currently due to #82)

**Comment by johanneskastl** [comment]
> Feel free to share a PR with all the changes you think are needed in current documentation.

Sure, once I got the hang of this and have gotten my hands dirty, I'll try to put together a PR.

**Comment by fmulero** [comment]
To change configuration database please follow this [comment](https://github.com/bitnami/bitnami-docker-openldap/issues/71#issuecomment-1026443653)

**Comment by johanneskastl** [comment]
> To change configuration database please follow this [comment](https://github.com/bitnami/bitnami-docker-openldap/issues/71#issuecomment-1026443653)

Sorry, I do not get what you mean.

If I understood you correctly, then changing the `olcsuffix` is not done when importing LIDFs, but I also cannot use `LDAP_ROOT` to do that (as it is being ignored if files are being imported).
So I would have to do that manually from within the container?

**Comment by fmulero** [comment]
Hi @johanneskastl. I revisit the code and the variable `LDAP_ROOT` is being taken into account to change the `olcsuffix` but you have to create the root object. Sorry about the misunderstood. I've just attached another [project](https://github.com/bitnami/bitnami-docker-openldap/files/8331912/gh-openldap-83-2.zip) with an example

**Comment by johanneskastl** [comment]
Fran, no need to apologize. Thanks for checking, then it looks like everything is logical.

I'll open up a PR to clarifiy that LDAP_ROOT can be used (and must be used) to set the proper olcSuffix, and add some more words on how the LDIF files should look like (aka what is working and what is not).

**Comment by fmulero** [comment]
Great!

**Comment by johanneskastl** [comment]
First draft is in #85, let me know what you think of it.

**Comment by johanneskastl** [comment]
Closed by #85
